_includes/footer.html

```diff
@@ -2,18 +2,20 @@
   <div class="contain">
     <img
       alt=""
+      role="presentation"
```

Comment:
logos and stuff like that dont need to be screen readable, and are in fact pretty annoying